src/commands/add/internal/input.ts

```diff
@@ -89,7 +89,7 @@ export async function getAndValidateServerUrl(target?: TargetJson) {
     target?.serverUrl
   )
 
-  return serverUrl
+  return serverUrl.replace(/\/$/, '')
```

Comment:
In the title, it says the leading slash should be removed, but `$` in regex will match the last character (trailing slash)

Reply:
aah, that's a bad title.  It should be trailing slash.  Good spot!